catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/event/SendEventTest.java

```diff
@@ -198,6 +199,7 @@ public void testIsAvailableRetryBackoff() throws Exception {
         assertFalse(available);
         assertNotEquals(lastPing, sendEvent.getLastPing());
         lastPing = sendEvent.getLastPing();
+        Thread.sleep(1);
```

Comment:
its looking for the time to be the same but it will pass even if it is broken if the test is fast enough